### PR TITLE
Add docs for opening an URL via automate.

### PR DESCRIPTION
**Pull request by martinpovolny**
preview the new article here: https://github.com/martinpovolny/guides/blob/automate_url_opener/automate_url_open.md

### CODE REVIEW

File: automate_url_open.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+### Opening custom URLs via Custom Buttons and Automate
+
+For a number of different entities users can create custom [toolbar
+buttons](ui/toolbars.md) that run an automate method on the particular entity,
+let that automate method calculate an URL and then have the UI open such URL in
```
Comment (mzazrivec) [inline automate_url_open.md]: `an URL` ➛ `a URL`

File: automate_url_open.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+a new windows.
```
Comment (mzazrivec) [inline automate_url_open.md]: `a new windows` ➛ `a new window`

File: automate_url_open.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      * data should be (or anything that sets `vm.remote_console_url` to something):
+
+```
+      vm = $evm.root['vm']
+      $evm.log(:info, "VM to launch SSH for is #{vm.hostnames[0]}")
+      vm.external_url = "https://www.google.com"
+
+```
+*important: use URL including the protocol (https) or the browser will ignore the request*
+*also disable URL pop-up blocking in your browser to make this work*
```
Comment (mzazrivec) [inline automate_url_open.md]: I'm afraid that visually, both these sentences will be merged into one (that will not look OK).

Comment (martinpovolny) [reply automate_url_open.md]: yes, see here: https://github.com/martinpovolny/guides/blob/automate_url_opener/automate_url_open.md

**Comment by martinpovolny** [comment]
@mzazrivec : Updated to address the issues.

**Comment by mzazrivec** [comment]
The docs here look good in my opinion and I also verified that the whole procedure works.

Not sure who can merge this though.

**Pull request merged**